JenkinsfileRT_dev

```diff
@@ -80,8 +76,8 @@ bc0.conda_packages = [
 ]
 bc0.pip_reqs_files = ['requirements-dev.txt']
 bc0.build_cmds = [
-    "pip install -e .[test,ephem]",
-    "pip install pytest-xdist pytest-sugar",
```

Comment:
Oh good. We need to cut back on sugar.

Reply:
That plugin was supposed to make the output pretty and also show the failure tracebacks right away in the output instead of you having to wait until the tests are finished to see what failed. But it doesn't work with Jenkins log capturing for some reason.

Reply:
So, no sugar tonight